ui/cypress/integration/queryBuilder.test.ts

```diff
@@ -1,10 +1,10 @@
 describe('query builder', () => {
   beforeEach(() => {
+    cy.OAuthLogin('test')
```

Comment:
At the beginning of each test, the user is not logged in.  If I tried to reach out to the Chronograf API while not logged in, I would be met with 403: Forbidden. I did try to log in by simply visiting the /login page and waiting for everything to happen. However, because the following line of tests always edits the Chronograf API with the user still waiting for authentication, it always ends with an error. Therefore I found it easier to log in as super admin and then edit Chronograf API. Also, wouldn't there be an issue if, let's say, the last test set tested the reader role, and the following test set tested super admin features? Since no one told OAuth2 Server that the user needs to be changed. I can change it so whenever the name parameter is not specified, log in as super admin, but that won't change how it works.